extender/main.go

```diff
@@ -108,53 +169,13 @@ func doKaniko(kind, baseimage string) {
 		if err != nil {
 			panic(err)
 		}
-	}
-
-	// Log the content of the Kaniko dir
-	fmt.Printf("Reading dir content of: %s\n", b.KanikoDir)
-	util.ReadFilesFromPath(b.KanikoDir)
-
-	// TODO: caching doesn't seem to be working at the moment. Need to investigate...
-	// Copy the tgz layer file to the Cache dir
-	srcPath := path.Join("/", b.LayerTarFileName)
-	dstPath := path.Join(b.CacheDir, b.LayerTarFileName)
-
-	// Ensure cache directory exists
-	fmt.Printf("Creating %s dir ...\n", b.CacheDir)
-	err = os.MkdirAll(b.CacheDir, os.ModePerm)
-	if err != nil {
-		panic(err)
-	}
 
-	fmt.Printf("Copy the %s file to the %s dir ...\n", srcPath, dstPath)
-	err = util.File(srcPath, dstPath)
-	if err != nil {
-		panic(err)
-	}
-
-	fmt.Printf("Extract the content of the tarball file %s under the cache %s\n", b.Opts.TarPath, b.Opts.CacheDir)
-	err = untar(dstPath, b.CacheDir)
-	if err != nil {
-		panic(err)
-	}
-
-	layerFiles, err := layerTarfiles()
-	if err != nil {
-		panic(err)
+		// TODO: next base image is the one we just built, we should use digest instead
+		current_base_image = b.Destination
 	}
 
+	// run buildpacks now
 	if kind == "build" {
-		// We're in "build" mode, untar layers to root filesystem: /
-		for _, layerFile := range layerFiles {
-			workingDirectory := "/"
-			tarPath := "/layers/kaniko/" + layerFile
```

Comment:
We should maybe avoid to hard code `kaniko` but pass it as parameter as it could be possible that another solution could be supported in the future (buildah, ...)